moped-etl/arcgis/components_to_agol.py

```diff
@@ -1,11 +1,20 @@
 #!/usr/bin/env python
 """Copies all Moped component records to ArcGIS Online (AGOL)"""
 # docker run -it --rm  --network host --env-file env_file -v ${PWD}:/app  moped-agol /bin/bash
```

Comment:
I went the docker way and not the virtual env way and had to update this to
`docker run -it --rm  --network host --env-file env_file  -v ${PWD}:/app  atddocker/atd-moped-etl-arcgis:production /bin/bash`